docs/contributing.rst

```diff
@@ -2,6 +2,44 @@ Contributing
 ############
 
 
+Setup your development environment
+==================================
+
+To prepare your system with Postgres and Redis please refer to the Installation guide.
+
+Prepare your project virtualenv:
+
+::
+
+    $ pip install virtualenv
+    $ virtualenv .venv
+    $ source .venv/bin/activate
+
+(deactivate when finished)
+
+Install packages as usual, for example install needed packages:
+
+::
+
+    $ pip install cliquet
+    $ pip install waitress
+    $ pip install tox
```

Comment:
you can actually install everything with the dev-requirement.txt file. `pip install -r dev-requirements.txt`